tests/components/api.spec.ts

```diff
@@ -112,8 +119,10 @@ describe('API', () => {
         transport.on.withArgs('async_file_transfer/data').onCall(0).callsFake((msg, cb) => cb({ payload }));
 
         transport.on.withArgs('async_file_transfer/done').onCall(0).callsFake((msg, cb) => {
+          clock.restore();
```

Comment:
Why do you have to do this? Can't you just to ```clock.tick(101);```

Reply:
If i tick, the timeout function will be triggered, rejecting the function call!